doc/encryption.md

```diff
@@ -106,18 +105,16 @@ protected void onCreate(Bundle savedInstanceState) {
  
     SQLiteDatabase.loadLibs(this);
  
-    // Create a DatastoreManager with encryption using 
+    // Get a DocumentStore instance with encryption using 
     // application internal storage path and a key
-    File path = getApplicationContext().getDir("datastores", MODE_PRIVATE);
+    File path = getApplicationContext().getDir("documentstores", MODE_PRIVATE);
     KeyProvider keyProvider = 
             new SimpleKeyProvider("testAKeyPasswordtestAKeyPassword".getBytes());
  
-    DatastoreManager manager = DatastoreManager.getInstance(path.getAbsolutePath());
- 
-    Datastore ds = null;
+    DocumentStore ds = null;
     try {
-        ds = manager.openDatastore("my_datastore", keyProvider);
-    } catch (DatastoreNotCreatedException e) {
+        ds = DocumentStore.getInstance(new File(path, "my_documentstore"), keyProvider);
+    } catch (DocumentStoreNotOpenedException e) {
         e.printStackTrace();
     }
         
```

Comment:
@emlaver I thought about the exception - the real fix is to regenerate the stack trace so that it has all of the correct class names, line numbers etc, but I think that's out of scope and I'm not even sure how to trigger this exception anyway